src/main/java/codeu/controller/MyProfileServlet.java

```diff
@@ -22,19 +27,64 @@
   	* Store class that gives access to Users.
   	*/
  	private UserStore userStore;
+    /** Store class that gives access to Conversations. */
+    private ConversationStore conversationStore;
+
+    /**
+     * Set up state for handling conversation-related requests. This method is only called when
+     * running in a server, not when running in a test.
+     */
+    @Override
+    public void init() throws ServletException {
+        super.init();
+        setUserStore(UserStore.getInstance());
+        setConversationStore(ConversationStore.getInstance());
+    }
+    /**
+    * Sets the UserStore used by this servlet. This function provides a common setup method for use
+    * by the test framework or the servlet's init() function.
+    */
+    void setUserStore(UserStore userStore) {
+        this.userStore = userStore;
+    }
+    
+    /**
+    * Sets the ConversationStore used by this servlet. This function provides a common setup method
+    * for use by the test framework or the servlet's init() function.
+    */
+    void setConversationStore(ConversationStore conversationStore) {
+        this.conversationStore = conversationStore;
+    }
+
+    @Override
+    public void doGet(HttpServletRequest request, HttpServletResponse response)
+        throws IOException, ServletException {
+
+            request.getRequestDispatcher("/WEB-INF/view/myprofile.jsp").forward(request, response);
+    }
+    @Override
+    public void doPost(HttpServletRequest request, HttpServletResponse response)
+    throws IOException, ServletException {
+        
+        String username = (String) request.getSession().getAttribute("user");
+        
+        User user = userStore.getUser(username);
+        
+        String conversationTitle = username;
```

Comment:
Hmm  so this is the user that is browsing through people's profile. Maybe the Session can have another attribute which records whose profile the current user is on?